mrbelvedereci/testresults/utils.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import get_object_or_404
+from mrbelvedereci.build.models import Build
+from mrbelvedereci.build.models import BuildFlow
+
+
+def find_buildflow(request, build_id, flow):
+    """ given a build_id and flow name, find a single BuildFlow (ala tests/ urls patterns). """
+    build = get_object_or_404(Build, id=build_id)
+
+    if not build.plan.public and not request.user.is_staff:
```

Comment:
<redacted_adminsite_url>/users/user/?is_staff__exact=1 - `is_staff` is internal users, `is_superuser` is admins. 

an auth helper for builds in general would be nice, and I know we have a ticket for using github access...